api/db-cdr/changelog-data/db.changelog-9.xml

```diff
@@ -18,12 +18,24 @@
       <column name="domain_id" type="STRING"/>
       <column name="name" type="STRING"/>
       <column name="description" type="STRING"/>
+      <column name="all_concept_count" type="NUMERIC"/>
```

Comment:
Are these changes to the other tables relevant to the PR? Wouldn't you also need to update the relevant CSV files?

Reply:
The CSV files already had these columns... the fact that they weren't in here before was an oversight.